modules/nf-core/kraken2/add/meta.yml

```diff
@@ -0,0 +1,54 @@
+name: kraken2_add
+description: Adds fasta files to a Kraken2 taxonomic database
+keywords:
+  - metagenomics
+  - db
+  - classification
+tools:
+  - kraken2:
+      description: "Kraken2 is a system for assigning taxonomic labels to short DNA sequences, usually obtained through metagenomic studies."
+      homepage: https://ccb.jhu.edu/software/kraken2/
+      documentation: https://github.com/DerrickWood/kraken2/wiki/Manual
+      tool_dev_url: "https://github.com/DerrickWood/kraken2"
+      doi: 10.1186/s13059-019-1891-0
+      licence: ["MIT"]
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. `[ id:'sample1', single_end:false ]`
+  - fasta:
+      type: file
+      description: fasta file that will be added to the database
+      pattern: "*.{fa,fasta,fna,ffn}"
```

Comment:
Are gzipped files allowed?

Reply:
> Are gzipped files allowed?

Not mentioned in the documentation, so I guess is not possible: [Kraken2: custom databases](https://github.com/DerrickWood/kraken2/blob/master/docs/MANUAL.markdown#custom-databases)